### ISSUE TITLE
Invalid highlighting for multiple post parameters in c++ pure virtual method

### ISSUE THREAD

**Issue by tbhartman** [jeff-hykin/cpp-textmate-grammar#301]
Multiple post parameters on c++ methods are not highlighted correctly if the method is pure virtual.  For example, "override" below is not highlighted:

```cpp
class Base
{
  public:
    virtual void method() const = 0;
}
class Derived : public Base
{
  public:
    virtual void method() const override = 0;
}
```

Expected:
![image](https://user-images.githubusercontent.com/896331/60981584-9fefcb80-a304-11e9-8281-e54c0f1f5e3f.png)

Actual:
![image](https://user-images.githubusercontent.com/896331/60981547-8d759200-a304-11e9-8ca2-770e3b21f00c.png)

observed in Visual Studio Code as microsoft/vscode#77107

**Comment by jeff-hykin** [comment]
@matter123 the core problem was fixed with `v1.12.19` just now, and should be sufficient for essentially every theme. But the scopes are still messed up a bit (you can see them in the tests that were updated with 082.cpp).

**Comment by jeff-hykin** [comment]
@tbhartman if you install the Better C++ Syntax extension, it should solve the issue now. 👍
